src/pyhf/__init__.py

```diff
@@ -27,6 +25,12 @@ def get_backend():
     return tensorlib, optimizer
 
 
+tensorlib = tensor.numpy_backend()
```

Comment:
why do we need this here and why didn't we need it before?

Reply:
`pyhf.optimizer = scipy_optimizer()` and `scipy_optimizer` from `opt_scipy` imports `OptimizerMixin` which needs `get_backend`.

Reply:
right but why do we need to set tensorlib here?

Reply:
ah sorry this was just moved down from above.. fine..